manifests/o/Obsidian/Obsidian/0.12.10/Obsidian.Obsidian.installer.yaml

```
@@ -0,0 +1,11 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Obsidian.Obsidian
+PackageVersion: 0.12.10
+Installers:
+- Architecture: x86
```

```suggestion
- Architecture: x64
```
Try it with `x64`; the `x86` VM might be bugging out.

I had no issues installing it on Windows Sandbox & VMware Workstation.